pkg/tfbridge/schema.go

```diff
@@ -1027,10 +1027,12 @@ func CleanTerraformSchema(tfs map[string]*schema.Schema) map[string]*schema.Sche
 func CoerceTerraformString(schType schema.ValueType, ps *SchemaInfo, stringValue string) (interface{}, error) {
 	// check for the override and use that over terraform if available
 	// we do this to ensure that we are following the explicit call to action of the override
+	// For now, we will only return nil when an override of the type is a boolean and there is no
```

Comment:
We should follow up with some investigation into whether or not this is actually safe. In particular, I'd like to know whether or not a non-existent property is treated identically to a property that exists and has the value `null`.